dashboard/config.py

```diff
@@ -1,70 +1,66 @@
-    SERVER_NAME = CONFIG("server_name", namespace="sso-dashboard", default="localhost:8000")
-    SESSION_COOKIE_NAME = SERVER_NAME + "_session"
+    SESSION_COOKIE_SAMESITE: str = os.environ.get("SSO-DASHBOARD_SESSION_COOKIE_SAMESITE", "lax")
+    SESSION_COOKIE_HTTPONLY: bool = os.environ.get("SSO-DASHBOARD_SESSION_COOKIE_HTTPONLY", "True") == "True"
 
-    S3_BUCKET = CONFIG("s3_bucket", namespace="sso-dashboard")
+    SECRET_KEY: str
+    SERVER_NAME: str = os.environ.get("SSO-DASHBOARD_SERVER_NAME", "localhost:8000")
+    SESSION_COOKIE_NAME: str
+    CDN: str
 
-    CDN = CONFIG(
-        "cdn",
-        namespace="sso-dashboard",
-        default="https://cdn.{SERVER_NAME}".format(SERVER_NAME=SERVER_NAME),
-    )
+    S3_BUCKET: str
+    FORBIDDEN_PAGE_PUBLIC_KEY: bytes
 
-    FORBIDDEN_PAGE_PUBLIC_KEY = base64.b64decode(CONFIG("forbidden_page_public_key", namespace="sso-dashboard"))
+    PREFERRED_URL_SCHEME: str = os.environ.get("SSO-DASHBOARD_PREFERRED_URL_SCHEME", "https")
+    REDIS_CONNECTOR: str
 
-    PREFERRED_URL_SCHEME = CONFIG("preferred_url_scheme", namespace="sso-dashboard", default="https")
-
-    REDIS_CONNECTOR = CONFIG("redis_connector", namespace="sso-dashboard")
+    def __init__(self):
+        self.SESSION_COOKIE_NAME = f"{self.SERVER_NAME}_session"
+        self.CDN = os.environ.get("SSO-DASHBOARD_CDN", f"https://cdn.{self.SERVER_NAME}")
+        self.REDIS_CONNECTOR = os.environ["SSO-DASHBOARD_REDIS_CONNECTOR"]
+        self.SECRET_KEY = os.environ["SSO-DASHBOARD_SECRET_KEY"]
+        self.S3_BUCKET = os.environ["SSO-DASHBOARD_S3_BUCKET"]
+        self.FORBIDDEN_PAGE_PUBLIC_KEY = base64.b64decode(os.environ["SSO-DASHBOARD_FORBIDDEN_PAGE_PUBLIC_KEY"])
 
 
-class OIDCConfig(object):
+class OIDC:
     """Convienience Object for returning required vars to flask."""
 
+    OIDC_DOMAIN: str
+    OIDC_CLIENT_ID: str
+    OIDC_CLIENT_SECRET: str
+    OIDC_REDIRECT_URI: str
+    LOGIN_URL: str
+
     def __init__(self):
         """General object initializer."""
-        CONFIG = get_config()
-        self.OIDC_DOMAIN = CONFIG("oidc_domain", namespace="sso-dashboard")
-        self.OIDC_CLIENT_ID = CONFIG("oidc_client_id", namespace="sso-dashboard")
-        self.OIDC_CLIENT_SECRET = CONFIG("oidc_client_secret", namespace="sso-dashboard")
-        self.LOGIN_URL = "https://{DOMAIN}/login?client={CLIENT_ID}".format(
-            DOMAIN=self.OIDC_DOMAIN, CLIENT_ID=self.OIDC_CLIENT_ID
+        self.OIDC_DOMAIN = os.environ["SSO-DASHBOARD_OIDC_DOMAIN"]
+        self.OIDC_CLIENT_ID = os.environ["SSO-DASHBOARD_OIDC_CLIENT_ID"]
+        self.OIDC_CLIENT_SECRET = os.environ["SSO-DASHBOARD_OIDC_CLIENT_SECRET"]
+        # Check for a prefixed environment variable, otherwise fallback to the
+        # unprefixed one.
+        self.OIDC_REDIRECT_URI = os.environ.get(
+            "SSO-DASHBOARD_OIDC_REDIRECT_URI", os.environ.get("OIDC_REDIRECT_URI", "https://sso.mozilla.com")
```

Comment:
Turns out I was conflating a couple of things here.

This config setting is specifically for `Flaks-pyoidc`. Though, I notice we don't use it anywhere anyways (the method which returns this doesn't make use of it).

Reply:
Done!